plugins/base/src/main/kotlin/renderers/DefaultRenderer.kt

```diff
@@ -141,6 +169,8 @@ abstract class DefaultRenderer<T>(
 
         renderPages(newRoot)
     }
+
+    override fun CoroutineScope.render(root: RootPageNode) = renderImpl(this, root)
```

Comment:
We should just `launch` and `join` somewhere there instead of in the generator IMO